Content.Server/Xenoarchaeology/XenoArtifacts/ArtifactSystem.Nodes.cs

```diff
@@ -48,7 +48,8 @@ private void GenerateNode(EntityUid artifact, ref List<ArtifactNode> uninitializ
         var node = uninitializedNodes.First();
         uninitializedNodes.Remove(node);
 
-        node.Id = _random.Next(0, 10000);
+        //random 5-digit number
+        node.Id = _random.Next(10000, 100000);
```

Comment:
Can't this result in duplicate IDs.

Reply:
I mean, it always could. I just ignored it because I didn't think it was worth the effort given the fact that there are like 12 nodes max and 90,000 different options.